python/mxnet/ndarray/random.py

```diff
@@ -518,3 +518,47 @@ def shuffle(data, **kwargs):
     <NDArray 2x3 @cpu(0)>
     """
     return _internal._shuffle(data, **kwargs)
+
+
+def randint(low=0, high=1, shape=_Null, dtype=_Null, ctx=None, out=None, **kwargs):
```

Comment:
should this also be included in the random distribution generator for the Symbol API here - https://github.com/apache/incubator-mxnet/blob/master/python/mxnet/symbol/random.py

FYI - the `randn` function - https://github.com/apache/incubator-mxnet/blob/master/python/mxnet/ndarray/random.py#L155 is defined only for NDArray and not for Symbol API.

Reply:
Point noted. I will include that in Symbol API.
Have created an issue for the randn function omission to get clarification.